test/Microsoft.DotNet.ShellShim.Tests/ShellShimMakerTests.cs

```diff
@@ -63,7 +65,7 @@ public void GivenAnExecutablePathItCanGenerateShimFile()
             var shellCommandName = nameof(ShellShimMakerTests) + Path.GetRandomFileName();
 
             shellShimMaker.CreateShim(
-                outputDll.FullName,
+                new FilePath(outputDll.FullName),
```

Comment:
Should you change MakeHelloWorldExecutableDll return a FilePath right away, instead of having to create one?